site/src/modules/apps/apps.ts

```diff
@@ -49,6 +61,73 @@ export const getTerminalHref = ({
 	}/terminal?${params}`;
 };
 
-export const openAppInNewWindow = (name: string, href: string) => {
+export const openAppInNewWindow = (href: string) => {
 	window.open(href, "_blank", "width=900,height=600");
```

Comment:
It looks like for `slim-window` apps we were adding the name before, now that we call `openAppInNewWindow` for those, should we add the name? `window.open(href, name, "width=900,height=600");`

Reply:
The second arg was never used to be a name tho https://developer.mozilla.org/en-US/docs/Web/API/Window/open but the target.

Reply:
True, but the that link says that the target can be:

> A string, without whitespace, specifying the [name](https://developer.mozilla.org/en-US/docs/Web/API/Window/name)

And the link to name says:

> The name of the window